### ISSUE TITLE
`r_wateralpha < 1` rendering issues

### ISSUE THREAD

**Issue by fhomolka** [fte-team/fteqw#210]
Games Tested: Quake 1
`r_wateralpha` has a few issues.

- At a distance, the brushes behind the liquid are not drawn unless the player or an entity casting light are near it. This causes the transparent, warping texture, to appear smeared.
Smeared:
![fte-20230809134112-0](https://github.com/fte-team/fteqw/assets/21170980/10ad92fe-bec5-4111-b9cd-9ea565ba8dcf)
Light in water:
![fte-20230809134119-0](https://github.com/fte-team/fteqw/assets/21170980/423f4f20-46b9-4fd0-ae18-146d07877ba9)

- At some parts of the map, it acts as straight up wallhacks. The specific case seems to be that there must be only the void between the space the player is in, and the loaded space
![clipboard](https://github.com/fte-team/fteqw/assets/21170980/7dfd718c-9804-4ab9-b3d9-e51f00cc05ab)

**Comment by shpuld** [comment]
afaik this happens when a map vis isn't compiled for transparent water.
there are versions of the maps that are, including the rereleased versions

correct behavior in this case would be to recognize if map has transparent
vis and turn off wateralpha but sounds unlikely that it's possible

**Comment by fhomolka** [comment]
> …

Ah yes, you're right!
Okay, the issue is different, then: FTE should probably warn if the map isn't water-vised, since the game is technically working correctly.

**Comment by eukara** [comment]
I am not sure there is a way to figure out if something is patched with watervis. How watervis works is like this, you take a .bsp and run the `vis` tool over it again, which is patched to ignore any water-prefixed-brushes during its calculation. It's making the map slightly less optimized so we can have transparent water not produce a hall of mirrors. There's no universal way to detect that has taken place. It just shakes up the portals within the maps.

**Comment by fhomolka** [comment]
Yeah, QSS has some detection. The message there says "Map **might not** be water-vised", which is fine. This will usually appear in older maps, since most modern map compilers will watervis. So, it's one of those "is it worth it" deals
